### DISCUSSION TITLE
How to access Github student discount when institutional mail has been discontinued?

### DISCUSSION THREAD

**Discussion by CodEZ47** [community/community#23493] [New to GitHub]
<p>I was trying to access the Github student developer pack using my institutional email. However, my school, which used to have institutional emails for every student has discontinued the service therefore rendering me incapable of being able to verify my institutional email address to use for discounts. And since my school is listed on the school listings Github requires I add an institutional email address… the institutional email address I can’t verify. What can I do about this inconvenience?</p>

**Comment by ettaboyle** [comment, marked as answer]
<p>Hi <a class="mention" href="/codez47">@CodEZ47</a> – Welcome to the GitHub Community Forums!</p>
<p>For student benefits and related conversations, we recommend visiting the <a href="https://github.com/orgs/github-community/discussions/9">Student Developer Pack FAQs post</a> as that may help answer your questions. If not, please use that Discussion to ask for assistance as that is the Education Community’s new home.</p>
